src/components/NavbarContainer.jsx

```diff
@@ -23,6 +23,12 @@ class NavbarContainer extends Component {
     this.setState({ showSearch: !showSearch });
   }
 
+  onLogoutClick = (e) => {
+    e.preventDefault();
+    localStorage.clear();
```

Comment:
Does this clear everything in your local-storage or just the item stored by redux persist? I believe there should be a better way of achieving our intended results.

Reply:
It completely remove everything stored in local storage for the website.